tests/integration_test.go

```diff
@@ -98,8 +99,8 @@ func TestXMRTaker_Discover(t *testing.T) {
 	_, err := bc.MakeOffer(xmrmakerProvideAmount, xmrmakerProvideAmount, exchangeRate)
 	require.NoError(t, err)
 
-	c := rpcclient.NewClient(defaultXMRTakerDaemonEndpoint)
-	providers, err := c.Discover(types.ProvidesXMR, defaultDiscoverTimeout)
```

Comment:
Just trying to make the tests easier to understand.  Bob was getting a `b` prefix, so I gave Alice an `a` prefix.